connector.rb

```diff
@@ -1,37 +1,2827 @@
+        field[:hint] = item_field[:hint]
+        field[:properties] = item_field[:properties]
+        field[:list_mode] = :static
+      else
+        error("Unsupported kind '#{type['kind']}'")
+      end
+      field[:toggle_field][:optional] = field[:optional] if field.has_key?(:toggle_field)
+      field
+    end,
+
+    apply_field_name_and_label: lambda do |connection, field, name_prefix, name|
+      field[:name] = if name_prefix.blank?
+                       name
+                     else
+                       "#{name_prefix}_#{name}"
+                     end
+      field[:label] = call('labelize', name, field['isDeprecated'])
+      field[:sticky] = true
+      toggle_field = field[:toggle_field]
+      if toggle_field.present?
+        toggle_field[:name] = field[:name]
+        toggle_field[:label] = field[:label]
+      end
+      field
+    end,
+
+    apply_field_description: lambda do |connection, field, description|
+      unless description.blank?
+        field[:hint] = call(
+          'format_field_hint',
+          connection,
+          description
+        )
+      end
+    end,
+
+    get_operation_type: lambda do |connection, operation_type|
+      schema = call('get_schema', connection)
+
+      # get operation type name
+      operation_type_name = schema.dig(
+        '__schema',
+        "#{operation_type}Type",
+        'name'
+      )
+
+      # get operation type
+      unless operation_type_name.blank?
+        call(
+          'get_schema_type',
+          connection,
+          nil,
+          'OBJECT',
+          operation_type_name
+        )
+      end
+    end,
+
+    create_field_for_input_field: lambda do |connection, report_problem, input_field, nesting_level|
+      field = call(
+        'create_field_for_type',
+        connection,
+        report_problem,
+        input_field['type'],
+        nesting_level
+      )
+      call(
+        'apply_field_name_and_label',
+        connection,
+        field,
+        nil,
+        input_field['name']
+      )
+      call(
+        'apply_field_description',
+        connection,
+        field,
+        input_field['description']
+      )
+      call(
+        'apply_field_default_value',
+        connection,
+        report_problem,
+        field,
+        input_field['defaultValue']
+      )
+      field
+    end,
+
+    format_field_hint: lambda do |connection, description|
+      if description.present?
+        description = description.strip
+        description = description.gsub('<p>', '<br>')
+        description = description.gsub('</p>', '<br>')
+        description = description.gsub(/\R/, '<br>')
+        description = description.gsub('\\n', '<br>')
+
+        allow_multi_paragraph_hint = false # call('get_allow_multi_paragraph_hint', connection)
+        first_break_index_one = description.index('<br>')
+        first_break_index_two = description.index('<br/>')
+        if !first_break_index_one.nil? && !first_break_index_two.nil?
```

Comment:
The multiline logic was copied from the GraphQL connector. There is no need to support multi `get_allow_multi_paragraph_hint`. I removed all code for `allow_multi_paragraph_hint`. 
It will be in the next commit.